### ISSUE TITLE
[public-1.10.0] Crash: server.dll + 0x122259

### ISSUE THREAD

**Issue by dysphie** [nmrih/source-game#970]
[Dumps #1](https://github.com/nmrih/source-game/files/1827649/Minidumps.zip)
[Dumps #2](https://github.com/nmrih/source-game/files/1980864/Minidumps2.zip)

What we know:
- Has never been seen happening in NMS
- Seems to occur when OnFullyOpen is fired
- Average recurrence is 3 to 4 times a week
- List of maps to ever experience this crash:
	- `nmo_miner_detour_v2` (15 crashes!) - [Dumps #3](https://github.com/nmrih/source-game/files/2479079/nmo_miner_detour_v2.zip)
	- `nmo_cold_black_y9v4` (4 crashes)
	- `nmo_penitentiary_v1` (2 crashes)
	- `nmo_brooklyn` (2 crashes)
	- `nmo_genex` (2 crashes)
	- `nmo_cabin` (2 crashes) - [Dumps #4](https://github.com/nmrih/source-game/files/2479105/cabin_crash_d3vwnkpd5xfg.zip)
	- `nmo_subside_v5` (1 crash) - [Dumps #5](https://github.com/nmrih/source-game/files/2479084/subside_crash_5jwowdwz46g6.zip)
	- `nmo_vertigo_v9` (1 crash)
	- `nmo_flowingdisaster_b3_mcg_v4` (1 crash)
	- `nmo_khafre_v7` (1 crash)
	- `pvp_colosseum_v7` (1 crash)
	- `nmo_zephyr` (1 crash)
	- `nmo_resident_evil_revelation2_v4` (1 crash)
	- `nmo_broadway` (1 crash)
	- `nmo_parkour_v5` (1 crash)

Tested builds:

- 1.09.6 [svn-r8572] for Windows (at server.dll@0x1c54ec)
- 1.10.0 [svn-r8886] for Windows (at server.dll@0x122259)

**Comment by Dysphie** [comment]
Still happens in public 1.10 as server.dll + 0x122259

**Comment by Dysphie** [comment]
Maps to ever crash on this function and the number of times:
nmo_miner_detour_v2 (15 crashes) [14 dumps](https://github.com/nmrih/source-game/files/2479079/nmo_miner_detour_v2.zip)
nmo_cold_black_y9v4 (4 crashes)
nmo_penitentiary_v1 (2 crashes)
nmo_brooklyn(2 crashes)
nmo_genex(2 crashes)
nmo_cabin (2 crashes)
nmo_subside_v5.bsp (1 crash) [1 dump](https://github.com/nmrih/source-game/files/2479084/subside_crash_5jwowdwz46g6.zip)
nmo_vertigo_v9 (1 crash)
nmo_flowingdisaster_b3_mcg_v4 (1 crash)
nmo_khafre_v7 (1 crash)
pvp_colosseum_v7 (1 crash)
nmo_zephyr (1 crash)
nmo_resident_evil_revelation2_v4 (1 crash)
nmo_broadway (1 crash)
nmo_parkour_v5 (1 crash)

If related to OnFullyOpen I/O, at least 2 maps on that list have none of it:
nmo_cabin, nmo_vertigo_v9

At the same time, nmo_miner_detour_v2 has 53 of them and Subside only 5, out of which this is the only one that can be interacted with:
```c
	"id" "228899"
	"classname" "func_door_rotating"
	connections
	{
		"OnFullyOpen" "!self,Lock,,0,-1"
	}```

**Comment by Pizhik** [comment]
Still happens
https://crash.limetech.org/noq4vdplmjz6
https://crash.limetech.org/oo2xfjiakfep
https://crash.limetech.org/gttlgk7muy5n

Installed stock mmsource-1.11.0-git1116-windows sourcemod-1.10.0-git6385-windows and Limetech's crash reporter.

**Comment by essbuh** [comment]
These crashes have likely all been fixed in code r4833 and should be included in 1.10.1.

**Comment by felis-catus** [comment]
Closing as resolved.
